src/plugins/recordTypes/movement/advancedSearch.js

```diff
@@ -0,0 +1,67 @@
+export default (configContext) => {
+  const {
+    OP_EQ,
+    OP_CONTAIN,
+    OP_RANGE,
+  } = configContext.searchOperators;
+
+  const {
+    defaultAdvancedSearchBooleanOp,
+    extensions,
+  } = configContext.config;
+
+  return {
+    op: defaultAdvancedSearchBooleanOp,
+    value: [
+      {
+        op: OP_EQ,
+        path: 'ns2:movements_common/currentLocation',
+      },
+      {
+        op: OP_EQ,
+        path: 'ns2:movements_anthropology/crate',
+      },
+      {
+        op: OP_RANGE,
+        path: 'ns2:movements_common/locationDate',
+      },
+      {
+        op: OP_EQ,
+        path: 'ns2:movements_anthropology/locationHandlers/locationHandler',
+      },
+      {
+        op: OP_EQ,
+        path: 'ns2:movements_common/movementMethods/movementMethod',
+      },
+      {
+        op: OP_EQ,
+        path: 'ns2:movements_common/reasonForMove',
+      },
+      {
```

Comment:
There are a bunch of fields below here that are hidden on the record editor. They should have been removed before, so let's do it now.